setup.py

```diff
@@ -32,7 +32,7 @@
 
 txt = (here / 'frozenlist' / '__init__.py').read_text('utf-8')
 try:
-    version = re.findall(r"^__version__ = '([^']+)'\r?$",
+    version = re.findall(r'^__version__ = "([^"]+)"\r?$',
```

Comment:
This whole hack could be replaced with adding `version = attr: frozenlist.__version__` to `setup.cfg`

Reply:
Totally